### PR TITLE
Remove report a problem form

### PR DESCRIPTION
**Pull request by lfdebrux**
https://trello.com/c/z63XpU1l/548-remove-is-there-anything-wrong-with-this-page-form

### CODE REVIEW

File: tests/main/views/test_feedback.py [hunk 1]
```diff
@@ -30,9 +29,3 @@ def test_google_gone_logs_problem(self, current_app, external_requests_post):
     def test_feedback_submission(self, external_requests_post):
         external_requests_post.return_value.status_code = 200
         self._check_success(self._post())
-
-    def test_feedback_form_has_csrf(self):
```
Comment (katstevens) [inline tests/main/views/test_feedback.py]: I wonder if we should return `404` at the `/feedback` view for now?

Comment (lfdebrux) [reply tests/main/views/test_feedback.py]: What would be the best way to do that, taking out the view altogether?

Comment (katstevens) [reply tests/main/views/test_feedback.py]: Strictly speaking it should be a `410 Gone`, as we're retiring the URL. We have a template for that already - let's keep the `send_feedback` view but replace all the code with `abort(410)`.

Comment (lfdebrux) [reply tests/main/views/test_feedback.py]: 👍 great that sounds like a plan

Comment (lfdebrux) [reply tests/main/views/test_feedback.py]: I've put together a commit to do this

**Pull request merged**